src/test/java/com/amazon/opendistro/elasticsearch/performanceanalyzer/collectors/StatsTests.java

```diff
@@ -89,6 +89,7 @@ public void testStats() throws Exception {
 
     private static void runInSerial(int iterateSize,  LinkedList<StatExceptionCode> exceptionCodeList, StatsCollector sc) {
         int i = 0;
+        sc.getCounters().clear();
```

Comment:
Can you provide some context for this change either here or add it to the description for this PR? I don't see any other changes in the code, so just wanted to know why we're changing a test without changing anything else.

Reply:
This was a patch added to the 1.7 branch to fix a unit test that was failing. More information can be found here https://github.com/opendistro-for-elasticsearch/performance-analyzer/pull/97.